source/test/TestApp/PlayFabEventTest.cpp

```diff
@@ -286,6 +335,15 @@ namespace PlayFabUnit
     {
         eventTestContext = nullptr;
         eventApi = nullptr;
+
+        if (testThreadPool.size() > 0)
+        {
+            for (auto& thread : testThreadPool)
```

Comment:
>testThreadPool [](start = 32, length = 14)

I don't understand why this thread pool needs to be static state. Wouldn't it be clearer if the test which used this threadPool (which is currently only 1) just explicitly joined all of its threads before returning? seems like it makes for a simpler understanding.

Unnecessarily sharing memory statically often leads to problems and is best avoided.